iocore/net/UnixNetVConnection.cc

```diff
@@ -1097,17 +1097,6 @@ UnixNetVConnection::acceptEvent(int event, Event *e)
   EThread *t    = (e == nullptr) ? this_ethread() : e->ethread;
   NetHandler *h = get_NetHandler(t);
 
-  MUTEX_TRY_LOCK(lock, h->mutex, t);
-  if (!lock.is_locked()) {
-    if (event == EVENT_NONE) {
-      t->schedule_in(this, HRTIME_MSECONDS(net_retry_delay));
-      return EVENT_DONE;
-    } else {
-      e->schedule_in(HRTIME_MSECONDS(net_retry_delay));
-      return EVENT_CONT;
-    }
-  }
-
```

Comment:
No longer needed because the lock is obtained in net_accept?

Reply:
Yes.